tests/sentry/integrations/msteams/test_notify_action.py

```diff
@@ -70,29 +67,50 @@ def test_applies_correctly(self):
         assert re.match(title_pattern, title_card["text"])
 
     @responses.activate
-    def test_applies_correctly_performance_issue(self):
-        event_data = load_data(
-            "transaction-n-plus-one",
-            timestamp=before_now(minutes=10),
-            fingerprint=[f"{GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value}-group1"],
-        )
-        perf_event_manager = EventManager(event_data)
-        perf_event_manager.normalize()
-        with override_options(
-            {
-                "performance.issues.all.problem-creation": 1.0,
-                "performance.issues.all.problem-detection": 1.0,
-                "performance.issues.n_plus_one_db.problem-creation": 1.0,
-            }
-        ), self.feature(
-            [
-                "organizations:performance-issues-ingest",
-                "projects:performance-suspect-spans-ingestion",
-            ]
-        ):
-            event = perf_event_manager.save(self.project.id)
+    @mock.patch(
+        "sentry.eventstore.models.GroupEvent.occurrence",
+        return_value=TEST_ISSUE_OCCURRENCE,
+        new_callable=mock.PropertyMock,
+    )
+    def test_applies_correctly_generic_issue(self, occurrence):
+        event = self.store_event(
+            data={"message": "Hello world", "level": "error"}, project_id=self.project.id
+        )
```

Comment:
We should probably change this to use https://github.com/getsentry/sentry/pull/42474/files#diff-adde26d90497aa681aad0bbaa83099f4524cf106f3e59ac4ef81f39d40db2e06R2434-R2445 once it's merged. Or maybe add a helper that allows us to quickly save a generic event